File: Math/squareroot/Python/square_root.py
def sqroot(x):
	if x == 0:
		return 0
		
	counter = 1
	while pow(counter, 2) <= x:
		counter += 1
	counter -= 1
	
	return counter
	

num = int(raw_input())
sqrt_num = sqroot(num)

print "The square root of the number %d is %d" % (num, sqrt_num)

File: Math/squareroot/C++/square_root.cpp
#include<iostream>
#include<math.h>
using namespace std;

int sqrt(int x)
{
  if(x==0) return 0;

  int counter=1;
  while(pow(counter,2)<=x){
    counter++;
  }
  counter--;
  return counter;
}
int main()
{
  int n;
  cout<<"Enter Number\n";
  cin>>n;
  cout<<"The square root of the number is: "<<sqrt(n)<<endl;
  return 0;
}
